.azure-pipelines.yml

```diff
@@ -253,22 +253,26 @@ jobs:
       # Check dependency compatibility
       pip check parsec[all]
     displayName: 'Install'
-  # Postgresql tests almost working, but _run function in tests/test_cli.py gives
-  # returncode of 1 instead of 0. Removed test_cli.py from this call as temporary fix
-  # doing tests only on tests/backend/*
-  # - bash: |
-  #     set -eux
-  #     py.test $(pytest.base_args) \
-  #       tests/backend --postgresql --runslow \
-  #       -n auto --max-worker-restart=0 -x \
-  #       --junitxml=test-results-postgresql.xml
-  #   displayName: 'Tests postgresql'
+  - bash: |
+      set -eux
+      py.test $(pytest.base_args) \
+        tests --runmountpoint --runslow \
+        -n auto --max-worker-restart=0 -x \
+        --junitxml=test-results-memory.xml
+    displayName: 'Tests memory'
   - bash: |
       set -eux
       py.test $(pytest.base_args) \
         tests --runmountpoint --runslow -m mountpoint -x \
         --junitxml=test-results-mountpoint.xml
     displayName: 'Tests mountpoint'
+  - bash: |
+      set -eux
+      py.test $(pytest.base_args) \
+        tests/backend tests/test_cli.py --postgresql --runslow \
```

Comment:
why testing also `tests/test_cli.py` with postgresql enabled ? (we normally only tests `tests/backend` with postgresql)

Reply:
I copied the lines from the linux tests (180th line). Should we fix it on both ?

Reply:
@vxgmichel @Max-7 Any idea why we enable postgresql in `test_cli.py` ?